src/classes/Manager.js

```diff
@@ -1,3 +1,11 @@
+
 export default class Manager {
-    
+  constructor(rooms, bookings) {
```

Comment:
Interesting construction of the Manager class. I passed through users so that the Manager would have access to all the users when manager needs to look up the users by name.